### PR TITLE
[CL-238] update typography

### PR DESCRIPTION
**Pull request by willmartian**
## Type of change

```
- [ ] Bug fix
- [x] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

- Update `bitTypography` to use new `headers` color
- Remove all font references to `Open Sans` and replace with `DM Sans`

- NO changes were made in this PR to base font size

## Code changes

- **file.ext:** Description of what was changed and why

## Screenshots

See Storybook

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: apps/browser/src/popup/scss/pages.scss [hunk 1]
```diff
@@ -306,7 +306,7 @@ app-fido2 {
       margin-top: 32px;
 
       .subtitle {
-        font-family: Open Sans;
+        font-family: DM Sans;
```
Comment (Hinton) [inline apps/browser/src/popup/scss/pages.scss:309]: To we need to use `"` here?
```suggestion
        font-family: "DM Sans";
```

File: .github/whitelist-capital-letters.txt [hunk 2]
```diff
@@ -5,16 +5,7 @@
...
-./libs/angular/src/scss/webfonts/Open_Sans-normal-300.woff
-./libs/angular/src/scss/webfonts/Open_Sans-normal-700.woff
-./libs/angular/src/scss/webfonts/Open_Sans-italic-300.woff
-./libs/angular/src/scss/webfonts/Open_Sans-italic-600.woff
-./libs/angular/src/scss/webfonts/Open_Sans-italic-800.woff
-./libs/angular/src/scss/webfonts/Open_Sans-italic-400.woff
-./libs/angular/src/scss/webfonts/Open_Sans-normal-600.woff
-./libs/angular/src/scss/webfonts/Open_Sans-normal-800.woff
-./libs/angular/src/scss/webfonts/Open_Sans-normal-400.woff
+./libs/angular/src/scss/webfonts/DMSans[opsz,wght].woff2
```
Comment (Hinton) [inline .github/whitelist-capital-letters.txt:8]: Could we just rename the file to be lowercase?

File: libs/components/src/typography/typography.directive.ts [hunk 3]
```diff
@@ -4,12 +4,12 @@ import { Directive, HostBinding, Input } from "@angular/core";
 type TypographyType = "h1" | "h2" | "h3" | "h4" | "h5" | "h6" | "body1" | "body2" | "helper";
 
 const styles: Record<TypographyType, string[]> = {
-  h1: ["tw-text-3xl", "tw-font-semibold"],
-  h2: ["tw-text-2xl", "tw-font-semibold"],
-  h3: ["tw-text-xl", "tw-font-semibold"],
-  h4: ["tw-text-lg", "tw-font-semibold"],
-  h5: ["tw-text-base", "tw-font-semibold"],
-  h6: ["tw-text-sm", "tw-font-semibold"],
+  h1: ["tw-text-3xl", "tw-font-semibold", "tw-text-headers"],
```
Comment (Hinton) [inline libs/components/src/typography/typography.directive.ts:7]: I can't seem to find `text-headers` in TW docs?

Comment (willmartian) [reply libs/components/src/typography/typography.directive.ts:7]: It is a new custom color in our config: https://github.com/bitwarden/clients/blob/ec37e5e4d3650079318a2a07ed94947d87feccf3/libs/components/tailwind.config.base.js#L62

Dark blue in light mode, white in dark mode

We missed adding it to the Storybook color docs.

File: .github/whitelist-capital-letters.txt (hunk 2 above)
Comment (willmartian) [reply .github/whitelist-capital-letters.txt:8]: Yes, good point.

**Review by cturnbull-bitwarden — APPROVED** [review]

File: libs/angular/src/scss/webfonts.css [hunk 4]
```diff
@@ -1,89 +1,7 @@
...
-  font-style: normal;
-  font-weight: 800;
-  font-display: auto;
-  src: url(webfonts/Open_Sans-normal-800.woff) format("woff");
-  unicode-range: U+0-10FFFF;
+  font-family: "DM Sans";
+  src:
+    url("webfonts/dm-sans[opsz,wght].woff2") format("woff2 supports variations"),
+    url("webfonts/dm-sans[opsz,wght].woff2") format("woff2-variations");
+  font-display: swap;
```
Comment (vleague2) [inline libs/angular/src/scss/webfonts.css:6]: Do we need to do the range definitions for stretch, style, and weight? (I'm still trying to read up on variable font implementations, so I'm not quite clear on all the settings)

**Review by cagonzalezcs — APPROVED** [review]

**Review by vleague2 — APPROVED** [review]

**Pull request merged**